_docs/ops/runbook/restoring-rds.md

```diff
@@ -29,13 +29,47 @@ The JSON contains the **database identifier** under the key `host` within the cr
 ## Restoring the Database
 Refer to the [RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html) for restoring a database.
 
+### Restoring from a Snapshot
+
 Prior to restoring, record the configuration settings:
+1. DB identifier
 1. Database publicly accessible from the internet (yes/no)
 1. Instance size (m4.large, etc.)
 1. Multi-zone (yes/no)
 1. VPC (dev, staging, etc.)
+1. Subnet
 1. Security Groups
 
+Launching the snapshot:
+- Log into the AWS Console and navigate to *RDS* > *Snapshots* and filter by <DB identifier>
+- Select the latest snapshot and click *Actions* > *Restore Snapshot*
+- In *Restore DB Instance*
+  - Under *Settings* > *DB Instance Identifier* enter a new DB identifier ie (`<DB identifier>-restore`)
```

Comment:
Is this how we're being more explicit about the naming?  Use the existing DB identifier name and add `-restore` to it?

That's fine by me, I just wanted to double check!

Reply:
@ccostino thanks for flagging I forgot to make a note. That's more of a placeholder for more discussion. Any thoughts?

Reply:
Honestly, that seems to be a good start to me and a reasonable convention!  It doesn't require a lot of thought and is easy to remember, so in tense situations that's one less thing to spend time worrying about and reasoning over even with written guidance.

Does anyone else have a strong opinion otherwise?